### ISSUE TITLE
Can't dump Starwing (SNES, PAL)

### ISSUE THREAD

**Issue by BenoitCaron** [sanni/cartreader#1028]
### OSCR firmware version

14.3

### OSCR hardware version

HW5 Rev5

### Attach OSCR_LOG.txt file here

OSCR HW5 V14.3

[+] Super Nintendo/SFC

[+] SNES/SFC cartridge

Searching database...
Checksum: 00FF
Header CRC32: AD9F2E43
Header CRC32: 546FF471
Header CRC32: A6932322
Header CRC32: B00AC917
Header CRC32: CB040CF6
Header CRC32: 86DAEEC6
Header CRC32: A5F599F1
Header CRC32: 5D1AFD5E
Header CRC32: 5101B422
Header CRC32: 5D9EED2B
Found

Name: STAR WING
Revision: 0
Type: LoROM SlowROM
ICs: ROM SuperFX
ROM Size: 1 MB (32 banks)
Save Size: 32 KB
Checksum: 00FF
Press Button...

[+] Read ROM

Saving to SNES/ROM/STAR WING/6/...
[*******************]
Checksum... 0214 != 00FF
Invalid Checksum
CRC32... C66B320D -> Not found
Press Button...

[+] Read ROM

Saving to SNES/ROM/STAR WING/7/...
[*******************]
Checksum... 02A6 != 00FF
Invalid Checksum
CRC32... 177D37D6 -> Not found

### System used

SNES/Super Famicom

### Describe the issue.

I have been trying to dump my PAL Starwing cartridges for ages now, expecting it would magically work with every new firmware release. I've thoroughly cleaned the contacts and inspected the PCB, but can't find anything wrong with it.
![IMG_3456](https://github.com/user-attachments/assets/aebe978b-87ad-447d-86ae-6b7923e17c02)
![IMG_3455](https://github.com/user-attachments/assets/3f40346a-6b6c-4012-8c3c-1be1559af39c)

**Comment by PsyK0p4T** [comment]
Oh... epoxy-style ! Is this coming from a PAL cartridge ?
Various Star Wing carts I've dumped until now were all using big PCBs, and dumped fine with OSCR. I've never tested one like yours.
![front](https://github.com/user-attachments/assets/7a4653f4-8c59-4aa5-972e-6b1a1035a796)
At least you got a rev0, confirmed by header and PCB label, so final CRC32 should be 865F1A71.

Have you tried to double clean the pins ? With a soft rubber for example. Also, using 5V ?

**Comment by BenoitCaron** [comment]
Yes the epoxy had me wondering at first, but there seems to be several revisions of Starwing, so it seems my cartridge is legit. I have cleaned the pins to the best of my ability, and I do use 5V. Again, this is the only cartridge I haven't been able to dump so far (tested every slot with every cartridge type multiple times).

**Comment by sanni** [comment]
Is there a way to tell what version of the PCB is inside the cartridge? Does the label differ from the normal version? It looks interesting.

**Comment by BenoitCaron** [comment]
Absolutely, here it is
![IMG_3498](https://github.com/user-attachments/assets/e0c702ce-e7a9-4823-8cfb-57b5d201a753)
